gamepanel/game.php

```diff
@@ -238,9 +253,10 @@ function titleBar()
 	{
 		$rightTop = '
 			<div class="titleBarRightSide">
-				<div>
-				<span class="gameTimeRemaining">'.$this->gameTimeRemaining().'</span></div>'.
-			'</div>';
+					<span class="gameTimeRemaining">'.$this->gameTimeRemaining().'</span>
+					<span class="gamePlayBeta">'.$this->gamePlayBeta().'</span>
+					<div style="clear:both"></div>

```

Comment:
this seems weird to me. why are we using a empty div to clear?

Reply:
This is how they position the items on both desktop and mobile with float. Ideally, I'd go with flex but that's going to require changes to the entire tiltle bar.